packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -41,7 +48,35 @@ export async function checkAndInstall(): Promise<Result<any, FxError>> {
 
     const failures: CheckFailure[] = [];
     const localEnvManager = new LocalEnvManager(VsCodeLogInstance, ExtTelemetry.reporter);
-    // TODO: LocalEnvManager deps
+
+    // Get project settings
+    if (!vscode.workspace.workspaceFolders) {
+      throw new UserError(
+        ExtensionErrors.NoWorkspaceError,
+        StringResources.vsc.handlers.noOpenWorkspace,
+        ExtensionSource
+      );
+    }
+    const workspaceFolder: vscode.WorkspaceFolder = vscode.workspace.workspaceFolders[0];
+    const workspacePath: string = workspaceFolder.uri.fsPath;
```

Comment:
Use `ext.workspaceUri` from "../extensionVariables"

Reply:
Got it!